eng/common/TestResources/New-TestResources.ps1

```diff
@@ -21,6 +21,9 @@ param (
     [Parameter(Mandatory = $true, Position = 0)]
     [string] $ServiceDirectory,
 
+    [Parameter()]
```

Comment:
cc @benbp 

You cannot edit these common files in the language repo instead you need to follow the https://github.com/Azure/azure-sdk-tools/blob/master/doc/common/common_engsys.md#workflow

Reply:
Thanks @weshaggard , I will follow the workflow and work with @benbp .